src/codeu/chat/util/CommandTokenizer.java

```diff
@@ -0,0 +1,67 @@
+package codeu.chat.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandTokenizer {
+
+	private String source;
+	private int currentIndex = 0;
+
+	public CommandTokenizer(String source) {
```

Comment:
Changing the order of the methods so that most methods on use methods above themselves. Not sure if there is a proper place to declare how the methods are order.